packages/react/src/components/EnablePushNotificationsBanner/EnablePushNotificationsBanner.tsx

```diff
@@ -35,18 +34,15 @@ export default function EnablePushNotificationsBanner() {
 
   const enablePushNotifications = () => {
     const subscribeUrl = path<string>(['webPush', 'config', 'subscribeUrl'], channels);
-    const url = axios.getUri({
-      url: subscribeUrl,
-      params: {
-        user_email: userEmail,
-        user_external_id: userExternalId,
-        background_color: theme.header.backgroundColor,
-        text_color: theme.header.textColor,
-      },
-    });
+
+    const url = new URL(subscribeUrl);
+    url.searchParams.append('user_email', userEmail);
+    url.searchParams.append('user_external_id', userExternalId);
+    url.searchParams.append('background_color', theme.header.backgroundColor);
+    url.searchParams.append('text_color', theme.header.textColor);
```

Comment:
Should we make this conditional, given the note in the PR description that axios omits undefineds, while URLSearchParam does not? The scarry thing is that there is no difference between `undefined` and `"undefined"`

```
new URLSearchParams({ foo: undefined, bar: 'undefined' }).toString() // foo=undefined&bar=undefined
```

```suggestion
    const { backgroundColor, textColor } = theme.header;
    
    const url = new URL(subscribeUrl);
    if (userEmail) url.searchParams.append('user_email', userEmail);
    if (userExternalId) url.searchParams.append('user_external_id', userExternalId);
    if (backgroundColor) url.searchParams.append('background_color', backgroundColor);
    if (textColor) url.searchParams.append('text_color', textColor);
```